### PR TITLE
Modified the README.md with updated installation instruction.

### PR DESCRIPTION
**Pull request by mhallsmoore**

### CODE REVIEW

**Comment by mhallsmoore** [comment]
@femtotrader, @ryankennedyio - Are you happy with the new README.md?

File: README.md [hunk 1]
```diff
@@ -21,67 +21,79 @@ The software is provided under a permissive "MIT" license (see below).
...
 
 * **Trading** - QSTrader will support live intraday trading using the Interactive Brokers API across a set of equities/ETFs.
 
-* **Performance Metrics** - QSTrader will support extensive strategy/portfolio performance measurement and equity curve visualisation via the Matplotlib and Seaborn visualisation libraries.
+# Installation and Example Usage
+
+QSTrader is in an extremely early alpha state at the moment and should only be used for exploratory backtesting research. The installation procedure is a little more involved than a standard Python package as it has not been added to the Python package repository yet.
 
-# Installation and Usage
+Ubuntu is the recommended platform on which to install QSTrader, but it will also work on Windows or Mac OSX under the Anaconda distribution ().
```
Comment (ryankennedyio) [inline README.md]: Unnecessary parentheses maybe?

**Comment by ryankennedyio** [comment]
Only other thing I might suggest is some info on how to analyse/view the .pkl files that are generated?

Otherwise, good good. Maybe some tips about generating and passing in the config file as a Click parameter, but don't think necessary at this stage, as anyone running it will be diving pretty deep into the code anyway.

**Comment by mhallsmoore** [comment]
Yeah, I think for the time being I'll ignore the Click parameter discussion. It might overcomplicate it for some users at this stage.

Hopefully once we package it up via PyPI then the installation will be far easier.

File: README.md [hunk 2]
```diff
@@ -21,67 +21,79 @@ The software is provided under a permissive "MIT" license (see below).
...
+source ~/venv/qstraderp3/bin/activate
 ```
 
-This will create a new virtual environment to install the packages into. Assuming you downloaded the QSTrader git repository into an example directory such as ```~/projects/qstrader/``` (change this directory below to wherever you installed QSTrader), then in order to install the packages you will need to run the following commands:
+At this point it is necessary to use pip to install QSTrader as a library and then manually install the requirements. The following steps will take some time as QSTrader relies on NumPy, SciPy, Pandas, Matplotlib as well as many other libraries and hence they will all need to compile:
 
 ```
-source ~/venv/qstrader/bin/activate
-pip install -r ~/projects/qstrader/requirements.txt
+pip install git+git://github.com/mhallsmoore/qstrader.git
```
Comment (femtotrader) [inline README.md]: I think

    $ pip install git+git://

should be

    $ pip install git+https://

**Pull request merged**